packages/ozone/src/util.ts

```diff
@@ -83,3 +83,17 @@ export const formatLabelerHeader = (parsed: ParsedLabelers): string => {
   )
   return parts.join(',')
 }
+
+export async function allFulfilled(
```

Comment:
This is fine, but there's a util in common `handleAllSettledErrors()` that does something similar.